crates/orchestrator/src/client.rs

```diff
@@ -193,7 +193,7 @@ impl OrchestratorClient {
     /// # Errors
     /// If we were unable to serialize the Libp2p data
     #[allow(clippy::type_complexity)]
-    pub async fn get_config_without_peer<K: SignatureKey>(
+    pub async fn config_without_peer<K: SignatureKey>(
```

Comment:
I think this should actually be `get_config_without_peer`. It's not a getter method, and I think `get` here is in the sense of `GET` as an HTTP request.

I would probably just revert the changes to the orchestrator, I think all the `get_` functions here are similar.

Reply:
Good catch! Yeah, I think it makes more sense to keep the `get` for these functions.